pages/linux/wodim.md

```diff
@@ -0,0 +1,17 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Caution when using this command.
+> There are many ways this command can be destructive, the following below is meant as a reminder of wodims syntax and not as the only source of documentation.
+
+- Display optical drives avaliable to wodim:
+
+`wodim --devices`
+
+- Burn file to disc in optical drive:
+
+`wodim -eject -tao speed=0 dev=/dev/{{optical_drive}} -v -data /{{file.iso}}`
```

Comment:
I have removed the speed switch since it seems easier to leave the speed up to the command to choose.
Removed the "/" no reason for it to be there.